docs/source/events/rendering.rst

```diff
@@ -0,0 +1,349 @@
+| ``enabled``           | If ``false` then use the old `Image` class.               |
++-----------------------+-----------------------------------------------------------+
+| ``tile_size``         | Size of render tiles to use for rending.                  |
++-----------------------+-----------------------------------------------------------+
+| ``log_path``          | Octree specific log file for debugging.                   |
++-----------------------+-----------------------------------------------------------+
+| ``loaders``           | Optional custom loaders, see below.                       |
++-----------------------+-----------------------------------------------------------+
+
+The ``loaders`` key lets you define and configure multiple
+:class:`~napari.components.experimental.chunk._pool.LoaderPool` pools. The
+key of each loader is the levels relative to the ideal level. In the above
+example configuration we define two loaders. The first with key "0" is for
+loading chunks at the ideal level or one above. While the second with key
+"2" will load chunks two above the ideal level or higher.
+
+Each loader uses the ``loader_defaults`` but you can override the
+``num_workers``, ``auto_sync_ms`` and ``delay_queue_ms`` values in
+each loader defined in ``loaders``.
+
+Multiple Loaders
+^^^^^^^^^^^^^^^^
+
+We allow multiple loaders to improve loading performance. There are a lot
+of different strategies one could use when loading chunks. For example,
+we tend to load chunks at a higher level prior to loading the chunks
+at the ideal level. This gets "coverage" on the screen quickly, and then
+the data can be refined by loading the ideal chunks.
+
+One consideration is during rapid movement of the camera it's easy to clog
+up the loader pool with workers loading chunks that have already moved out
+of view. The
+:class:`~napari.components.experimental.chunk._delay_queue.DelayQueue` was
+created to help with this problem.
+
+While we can't cancel a load if a worker as started working on it, we can
+trivially cancel loads that are still in our delay queue. If the chunk goes
+out of view, we cancel the load. If the user pauses for a bit, we initiate
+the loads.
+
+With multiple loaders we can delay the ideal chunks, but we can configure
+zero delay for the higher levels. A single chunk from two levels up will
+cover 16 ideal chunks. So immediately loading them is a good way to get
+data on the screen quickly. When the camera stops moving the
+:class:`~napari.components.experimental.chunk._pool.LoaderPool` for the
+ideal layer will often be empty. So all of those workers can immediately
+start loading the ideal chunks.
+
+The ability to have multiple loaders was only recently added. We still need
+to experiment to figure out the best configuration. And figure out how that
+configuration needs to vary based on latency of the data or other
+considerations.
+
+Future Work: Extending TextureAtlas2D
```

Comment:
I can't remember if I saw this elsewhere earlier, or if it's here and i'm just missing it, but should we add a section here like `Known Octree Limitations` and talk about how contrast limits, opacity, colormaps, and different blending modes aren't supported yet. Also any known dtypes that aren't supported. I haven't tested with multiscale timeseries, but if you know that doesn't work could add that too.

Reply:
I put in several new sections for each of these. Giving my assessment of where we might want to go. Although a fresh set of ideas might have solutions I'm not thinking of.